pkg/controller/ingress/ingress_controller.go

```diff
@@ -358,6 +437,13 @@ func (r *ReconcileIngress) delete(instance *extensionsv1beta1.Ingress) (*extensi
 		zap.String("stackName", instance.ObjectMeta.Name),
 		zap.String("status", *stack.StackStatus),
 	)
+
+	err = r.verifyAndUpdateASG(instance, true)
```

Comment:
What happens in this scenario, Delete stack completes but the ASG update does not finish and requeue ?

Reply:
Not sure I get it. This call is happening before calling DeleteStack and i believe they are blocking calls.

Reply:
Ah, my bad, your right